openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -198,9 +206,11 @@ This is an example of a Wallet Instance Attestation:
 
 * The Credential Issuer MUST publish a mapping of every Credential Type it supports to a scope value.
 
-# OpenID for Verifiable Presentations
+# OpenID for Verifiable Presentations profile for IETF SD-JWT VC over HTTPS
+
+The requirements for the Wallet and the Verifier, unless specified otherwise:
```

Comment:
```suggestion
Requirements for the Wallet and the Verifier, unless specified otherwise:
```
(Do we need the "unless specified otherwise" here?)

Reply:
```suggestion
Requirements for the Wallet and the Verifier:
```
we probably don't